src/Microsoft.Dnx.Compilation/CompilationEngine.cs

```diff
@@ -59,7 +59,9 @@ public LibraryExporter CreateProjectExporter(Project project, FrameworkName targ
             {
                 Project = project,
                 RuntimeIdentifiers = _context.RuntimeEnvironment.GetAllRuntimeIdentifiers(),
-                TargetFramework = _context.ApplicationEnvironment.RuntimeFramework
+                TargetFramework = _context.ApplicationEnvironment.RuntimeFramework,
+                SkipRuntimeMinVersionCheck = true,
+                RuntimeEnvironment = _context.RuntimeEnvironment
```

Comment:
If runtime version check is skipped here, the `RuntimeEnvironment` instance is not required. Do we have to set it?